src/couch_mrview/src/couch_mrview_test_util.erl

```diff
@@ -53,7 +53,8 @@ ddoc(changes) ->
     couch_doc:from_json_obj({[
         {<<"_id">>, <<"_design/bar">>},
         {<<"options">>, {[
-            {<<"seq_indexed">>, true}
+            {<<"seq_indexed">>, true},
+            {<<"keyseq_indexed">>, true}
```

Comment:
It is valid to have only seq btree enabled on a view, i.e. all tests should pass even with key-seq btree not built. If anything we should expand the tests to assert both variants.

Can you explain, please, why do you think changing a scope of `couch_mrview_changes_since_tests.erl` to test `key-seq` instead of `seq` is the right fix?